interp/constrextern.ml

```diff
@@ -444,7 +447,7 @@ let rec extern_cases_pattern_in_scope (custom,scopes as allscopes) vars pat =
           (insert_pat_alias ?loc (insert_pat_delimiters ?loc (CAst.make ?loc @@ CPatPrim p) key) na)
   with No_match ->
     try
-      if !Flags.in_debugger || !Flags.raw_print || !print_no_symbol then raise No_match;
+      if !Flags.in_debugger || !Flags.raw_print || not !print_primitive_token then raise No_match;
```

Comment:
I see that `print_no_symbol` is changed to `not !print_primitive_token` even in the non-prim-token case. So as to understand your semantics, could you give details about the intended effect of the four different combinations of the two options (and in particular whether they are intended to be strictly disjoint or if one has an impact on the other)?

Reply:
If I understood the code correctly, the two options are now orthogonal.

I did prefer the default to true (rather than false) hence the missing `_no_` in the option name (and the `not` in its tests).